dbfit-java/core/src/test/java/dbfit/util/DbParameterAccessorTest.java

```diff
@@ -0,0 +1,43 @@
+package dbfit.util;
+
+import dbfit.fixture.StatementExecution;
+import static dbfit.util.Direction.INPUT_OUTPUT;
+import org.junit.Test;
+import static org.mockito.Mockito.*;
+import static java.sql.Types.*;
+
+public class DbParameterAccessorTest {
+
+    int sqlType = VARCHAR;
+    Class<?> javaType = String.class;
+    String inputValue = "The input value";
+    int position = 1;
+    Integer outputValue = 5;
+    String userDefinedTypeName = "whatever";
+    TypeTransformer inputTransformer = mock(TypeTransformer.class);
+    TypeTransformerFactory inputTransformerFactory = mock(TypeTransformerFactory.class);
+    StatementExecution statement = mock(StatementExecution.class);
+    DbParameterAccessor dbpa = new DbParameterAccessor("dummy", INPUT_OUTPUT, sqlType,
+                                                       userDefinedTypeName, javaType, position, inputTransformerFactory);
+
+    public DbParameterAccessorTest() throws Exception {
+        dbpa.bindTo(statement, position);
+    }
+
+    @Test
+    public void inputObjectIsTransformedBeforeJdbcBindingWhenTransformerSpecifierTest() throws Exception {
+
```

Comment:
I'm easy. I'll remove them.

I'm not sure about the line after the class declaration though. We seem to have no standard. Some are followed by a blank and some not, even some of the newer classes added.